src/client/App.js

```diff
@@ -1,33 +1,53 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import { ConnectedRouter } from 'connected-react-router';
-import { Switch, Route } from 'react-router-dom';
-import PrivateRoute from './components/PrivateRoute/PrivateRoute';
-import GuestRoute from './components/GuestRoute/GuestRoute';
-import CheckIfLoggedIn from './components/CheckIfLoggedIn/CheckIfLoggedIn';
+import React, { useEffect, useState } from 'react';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import { Login, Registro, Home } from './pages/';
+import usuarioContexto from './state/state';
 
-import LoginPage from './pages/Login/Login';
-import RegisterPage from './pages/Register/Register';
-import DashboardPage from './pages/Dashboard/Dashboard';
-import PageNotFound from './pages/NotFound/NotFound';
+export default function App() {
+  const [user, setUser] = useState({ nombre: '', uid: '', islogged: false });
```

Comment:
This is instantiating the state with an empty object. It would be a better practice to have `null` or `undefined` in the state if there is no logged in user.